qucs-doc/report/EKV26/Makefile.am

```diff
@@ -29,6 +29,8 @@ EXTRA_DIST = $(SOURCES)
 SOURCES = EKV26.tex content.tex
 TARGETS = EKV26.pdf
 
+export TEXINPUTS=$(srcdir):$(srcdir)/..:
```

Comment:
yes, now is clearer. Fully agree that what we would like to do is argument passing but as you said unfortunately we can't do that.

The C compiler is invoked as `$(CC) $(CCFLAGS)` by `make`, you do not have to explicitly write a rule for this in `Makefile.am`. I'd like to write a line like `tutorial_SOURCES = tutorial.tex` (etc.) and having automake automatically  prepared the command line needed, but that's not possible.  Using the `export` avoid typing things explicitly, as said, in the same way as you do not type `$(CC) $(CCFLAGS)`  in `Makefile.am`.

> no. global variables are more cluttered. seriously.

Care to explain, seriously? There is less text that does not add much to understanding what the rule is doing, so there is less clutter in `Makefile.am`. The shell in which the command is run has the same number of environment variables in both cases. The end effect is the same.

> exporting is bad.

yes, you said that already, but did not explain why. Repeating without an explanation sounds like the cargo cult you were mentioning. I tried to explain why I think it's not bad; please explain which bad things will happen using the `export` here.

To me all this, until now, looks like a difference of style only.